src/nIsForName.js

```diff
@@ -22,7 +22,7 @@
  * @return {string[]}
  */
 function nIsForName(names) {
-  // write code here
+  return names.filter(el => el.includes('N'));
```

Comment:
This solution is nice but very weak. It can have unexpected behavior if the name is like 'ARNOLD'. We'll add test to cover this case but for the moment you have to fix solution to let it work only for names which part starts with `N`